### PR TITLE
Redshift dependent SED

### PR DESCRIPTION
**Pull request by jucordero**
Implemented a redshift dependent model for the SED, using a linear combination of template SED with coefficients drawn from a Dirichlet distribution to address #2

### CODE REVIEW

File: forward/sed.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from astropy.io import fits
+
+def alphas(z, alpha_0, alpha_1, z1 = 1):
+    """ For a given set of alpha_0, alpha_1 values, return the alpha values for the
+    Dirichlet distribution at the given redshift z, evolving from z=0 to z=z1 (z1 = 1 by default)"""
+
+    return np.power(alpha_0, 1-(z/z1)) * np.power(alpha_1, (z/z1))
```
Comment (itrharrison) [inline forward/sed.py]: Can you change this to e.g.
`dirichlet_alphas = np.power(alpha_0, 1-(z/z1)) * np.power(alpha_1, (z/z1))`
`return dirichlet_alphas`
as this makes debugging easier.

File: forward/sed.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+def f_lambda(alphas, template_file, ext = 1):
+    """ For a given set of alpha, return the c_i weighted SED """
+    ci = np.random.dirichlet(alphas)
```
Comment (itrharrison) [inline forward/sed.py]: `ci` needs to have dimensions of number of templates.
Here, has dimensions of number of redshifts passed to `alphas`.

**Review by itrharrison — CHANGES_REQUESTED** [review]

File: forward/sed.py (hunk 1 above)
Comment (jucordero) [reply forward/sed.py]: Done, should appear in new commits

File: forward/sed.py (hunk 2 above)
Comment (jucordero) [reply forward/sed.py]: It now has `ci.shape = (nz, nt)`, number of redshifts times number of templates.
The whole function now should return an array of dimensions nz, nlambda

**Review by itrharrison — APPROVED** [review]

**Pull request merged**